.eslintrc.cjs

```diff
@@ -73,23 +73,23 @@ module.exports = {
     {
       files: ["**/*.{js,mjs,cjs}"],
       parserOptions: {
-          ecmaVersion: 13, // ES2022 - https://eslint.org/docs/latest/use/configure/language-options#specifying-environments
+        ecmaVersion: 13, // ES2022 - https://eslint.org/docs/latest/use/configure/language-options#specifying-environments
```

Comment:
Not entirely sure why this is prettifying, but CI fails without it.

Reply:
See https://github.com/connectrpc/connect-es/pull/922. Best to merge 922 to main, and merge that in here.